content/x-ray/x-ray.md

```diff
@@ -26,3 +26,17 @@ Click on the image to zoom
 {{% /notice %}}
 
 
+#### Challenge
+**Append the  _?slow=true_ query param to the service endpoint in the browser and diagnose the increase in latency.**
+
+**HINT:** How were the microservices performing before you added this query parameter?
```

Comment:
What if we ask, what is the latency of backend application before you added this query parameter?
this would make it easier to find relevant info. Also is their x-ray documentation on how to diagnose this info?